spec/requests/admin/users_spec.rb

```diff
@@ -10,44 +10,55 @@
 
     context 'ログインしている場合' do
       context 'ユーザが管理者の場合' do
-        context 'クエリパラメータにpageがない場合' do
-          xit 'ユーザー取得APIにoffset=0パラメータをつけて呼び出すこと' do
+        context 'ユーザー取得の関数の実行時に例外が返ってきた場合' do
+          xit 'ホーム画面にリダイレクトして、取得に失敗した旨をトーストメッセージで表示' do
             # TODO: specの内容を作成する
           end
```

Comment:
そういえば何度か見かけつつアドバイスを入れるタイミングをなくしていたのだけれど、

`describe` `context` `it` は「ブロックを与えない」だけでも pending 扱いになりますよ。  
明示的に書きたいなら止めませんが。

```suggestion
          it 'ホーム画面にリダイレクトして、取得に失敗した旨をトーストメッセージで表示'
```